.eslintrc.json

```diff
@@ -30,10 +30,33 @@
         "/coverage/**"
     ],
     "rules": {
-        "max-statements-per-line": ["error", { "max": 1 }],
-        "simple-import-sort/imports": "error",
-        "simple-import-sort/exports": "error",
+        // eslint
+        "dot-notation": "error",
```

Comment:
This one may actually be a formatting rule, but neither dprint nor prettier do conversion from `foo["bar"]` to `foo.bar` or vice-versa.